### ISSUE TITLE
Allow Room Host To Abort Game

### ISSUE THREAD

**Issue by MkTr** [tetrio/issues#976]
### Checklist

- [X] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this feature is not already requested.
- [X] I am using the **newest version** of TETR.IO.
- [X] I am certain this is a **feature request**, not a bug.
- [X] I've **thought this feature through** properly.

### Is your feature request related to a problem? Please describe.

So recently the S And Below room autohost disappeared, and after a while I got to be the host
as a joke I set the garbage multiplier to 10 for one round, but accidentally set it to 0 on the next round and nobody received any garbage
the lobby spent the next 5 minutes spectating the six dumbasses that didn't catch on while my dumbass left the room to leave everyone pleading with the new host to change the settings (which for the love of god they did)

### Describe the solution you'd like

basically this was the funniest thing that's happened in tetrio and while it _would_ be hilarious if it happened a second time I'd like it if the host had the power to abort a game off at any point.

### Describe alternatives you've considered

The problem with this is that a salty host can just shut off the game after losing (though, arguably, a salty host can screw with the settings to make the game unplayable), so I understand if you disagree with the feature.

**Comment by MkTr** [comment]
Oh yeah, maybe it would be better to keep the ability to end the game "hidden" as a chat command instead of a button, so it's not as easy to do out of rage/mistake.
